x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/use_columns.tsx

```diff
@@ -388,7 +393,7 @@ export const useMonitoringColumns = ({ hasPermissions }: ColumnsProps): TableCol
         ),
         render: (value: DurationMetric | undefined) => (
           <EuiText data-test-subj="gap" size="s">
-            {value != null ? moment.duration(value, 'seconds').humanize() : getEmptyTagValue()}
+            {value != null ? durationFormat.convert(value) : getEmptyTagValue()}
```

Comment:
What do you think about replacing custom formatters in other places across Security Solution? Also, it would make sense to use `fieldFormats` for other field types (number, time, percentage, etc.).

Reply:
I've created a discussing issue [#142720](https://github.com/elastic/kibana/issues/142720). This discussion goes far beyond the current PR's scope.